xcode/Subconscious/Shared/Components/FirstRun/FirstRunCreateSphereView.swift

```diff
@@ -9,25 +9,44 @@ import SwiftUI
 
 struct FirstRunCreateSphereView: View {
     @ObservedObject var app: Store<AppModel>
+    @Environment(\.colorScheme) var colorScheme
+    
+    private var bgGradient: LinearGradient {
+        switch colorScheme {
+        case .dark:
+            return Color.bgGradientDark
+        default:
+            return Color.bgGradientLight
+        }
+    }
+
+    private var shadow: Color {
+        switch colorScheme {
+        case .dark:
+            return .brandBgPurple
+        default:
+            return .brandMarkPurple
+        }
+    }
```

Comment:
This fragment is repeated a lot, not sure if there's a clever way to extract it?